Assets/Modules/Enemies/EnemyKillController.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyKillController : MonoBehaviour
```

Comment:
I would call this class the same as `PlayerLifeController` not killController. The reason is if two classes are doing the same job for different entities (even though the logic is different), we should keep their name as consistent as possible.